party-role/src/test/resources/application.yaml

```diff
@@ -0,0 +1,45 @@
+micronaut:
+
+  server:
+    port: 8632
+
+  caches:
```

Comment:
The same changes in application properties should be done for tests (check account-module, for example)